.gitignore

```diff
@@ -51,4 +51,10 @@ sitemap-*.xml
 **/public/worker-*.js
 **/public/sw.js.map
 **/public/workbox-*.js.map
-**/public/worker-*.js.map
\ No newline at end of file
+**/public/worker-*.js.map
+*.zip
+*.yml
+*.gz
+*.lock
+*.lock
+yarn.lock
```

Comment:
Why did you add all of these to the `gitignore`? We want the `yarn.lock` file committed (https://stackoverflow.com/a/39992365).

Reply:
Possibly my bad, I must've not noticed that yarn.lock was one of the files to be committed in the vs code version control panel. I offhandedly said to only commit the file he had changed. Oopsies